vxquery-core/src/main/javacc/xquery-grammar.jj

```diff
@@ -97,15 +97,15 @@ public class XMLQuery {
 
 PARSER_END(XMLQuery)
 
-	
+    
```

Comment:
Christina did a search and replace for tabs in this file. I think we can allow the whitespace changes for this use case.